app/decorators/sponsor_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class SponsorDecorator < Draper::Decorator
+  delegate_all
+
+  def contact_info
+    [ member_contact_details, contact_full_name, email ].flatten.delete_if(&:empty?).join("<br/>").html_safe
```

Comment:
right before or after the delete_if method, please also add a `compact` - there have been issues with methods being called on nil object